docs/notebooks/datasets/ws-353.txt

```diff
@@ -0,0 +1,353 @@
+love	sex	6.77

```

Comment:
This dataset is already in Gensim in https://github.com/parulsethi/gensim/blob/develop/gensim/test/test_data/wordsim353.tsv Please use it there